### PR TITLE
Appoint a rep e2e tests

### PR DESCRIPTION
**Pull request by cosu419**
Added E2E tests for navigating through two form flows.

## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to TeamSites and delete the rest of this section)

Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No

## Summary

E2E tests for two flows:

- 21-22 form, authenticated where user is a veteran
- 22-22a form, unauthenticated where user is a non-veteran claimant

In the former, backwards navigation is simulated. In the latter, the submission process is simulated (user cannot nav backwards from here). 

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/92156

## Acceptance criteria

- [x] End to end tests have been written for Appoint a Representative MVP

- [x] All tests passing

### Quality Assurance & Testing
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed

### CODE REVIEW

**Review by holdenhinkle — COMMENTED** [review]
Please add an axe check to each page.

I think we should add a number of checks to the review page when we have time after staging review, just to verify that what the user selected through the form flow is displaying as expected on the user review page.

**Comment by oddball-lindsay** [comment]
@holdenhinkle we did Axe Testing as part of the Pre-Staging Review requirement for accessibility testing: https://github.com/department-of-veterans-affairs/va.gov-team/issues/97463

Does that cover your request for axe checks on each page?

File: src/applications/representative-appoint/tests/e2e/navigation/2122a.cypress.spec.js [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+    });
+
+    it('navigates through the flow successfully', () => {
+      cy.visit(h.ROOT);
+
+      // INTRO
+      h.verifyUrl(ROUTES.INTRO);
+      cy.injectAxeThenAxeCheck();
+
+      cy.injectAxeThenAxeCheck();
```
Comment (holdenhinkle) [inline src/applications/representative-appoint/tests/e2e/navigation/2122a.cypress.spec.js:25]: You can delete this axe check - `cy.injectAxeThenAxeCheck();` - no need to do two checks consecutively.

**Review by holdenhinkle — COMMENTED** [review]
Looks good! Just one minor thing...

**Review by holdenhinkle — DISMISSED** [review]

**Review by holdenhinkle — APPROVED** [review]

**Pull request merged**